SwipeView causes Java.Lang.IllegalArgumentException: Cannot add a null child view to a ViewGroup

### Description

I'm using a `SwipeView` (inside a `CollectionView`, but I don't know if it is important or not).
Sometimes (rarely) I get an exception when starting the swipe.
I don't have a repro, but I have the full stacktrace.

```
[mono-rt] [ERROR] FATAL UNHANDLED EXCEPTION: Java.Lang.IllegalArgumentException: Cannot add a null child view to a ViewGroup
[mono-rt]    at Java.Interop.JniEnvironment.InstanceMethods.CallNonvirtualVoidMethod(JniObjectReference instance, JniObjectReference type, JniMethodInfo method, JniArgumentValue* args) in /Users/runner/work/1/s/xamarin-android/external/Java.Interop/src/Java.Interop/obj/Release/net8.0/JniEnvironment.g.cs:line 20831
[mono-rt]    at Java.Interop.JniPeerMembers.JniInstanceMethods.InvokeVirtualVoidMethod(String encodedMember, IJavaPeerable self, JniArgumentValue* parameters) in /Users/runner/work/1/s/xamarin-android/external/Java.Interop/src/Java.Interop/Java.Interop/JniPeerMembers.JniInstanceMethods_Invoke.cs:line 75
[mono-rt]    at Android.Views.ViewGroup.AddView(View child) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/obj/Release/net9.0/android-35/mcw/Android.Views.ViewGroup.cs:line 2129
[mono-rt]    at Microsoft.Maui.Handlers.SwipeItemViewHandler.UpdateContent() in /_/src/Core/src/Handlers/SwipeItemView/SwipeItemViewHandler.Android.cs:line 41
[mono-rt]    at Microsoft.Maui.Handlers.SwipeItemViewHandler.MapContent(ISwipeItemViewHandler handler, ISwipeItemView page) in /_/src/Core/src/Handlers/SwipeItemView/SwipeItemViewHandler.Android.cs:line 47
[mono-rt]    at Microsoft.Maui.PropertyMapper`2.<>c__DisplayClass5_0[[Microsoft.Maui.ISwipeItemView, Microsoft.Maui, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null],[Microsoft.Maui.Handlers.ISwipeItemViewHandler, Microsoft.Maui, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]].<Add>b__0(IElementHandler h, IElement v) in /_/src/Core/src/PropertyMapper.cs:line 172
[mono-rt]    at Microsoft.Maui.PropertyMapper.UpdatePropertyCore(String key, IElementHandler viewHandler, IElement virtualView) in /_/src/Core/src/PropertyMapper.cs:line 47
[mono-rt]    at Microsoft.Maui.PropertyMapper.UpdateProperties(IElementHandler viewHandler, IElement virtualView) in /_/src/Core/src/PropertyMapper.cs:line 82
[mono-rt]    at Microsoft.Maui.Handlers.ElementHandler.SetVirtualView(IElement view) in /_/src/Core/src/Handlers/Element/ElementHandler.cs:line 79
[mono-rt]    at Microsoft.Maui.Handlers.ViewHandler`2[[Microsoft.Maui.ISwipeItemView, Microsoft.Maui, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null],[Microsoft.Maui.Platform.ContentViewGroup, Microsoft.Maui, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]].SetVirtualView(IView view) in /_/src/Core/src/Handlers/View/ViewHandlerOfT.cs:line 53
[mono-rt]    at Microsoft.Maui.Handlers.SwipeItemViewHandler.SetVirtualView(IView view) in /_/src/Core/src/Handlers/SwipeItemView/SwipeItemViewHandler.Android.cs:line 25
[mono-rt]    at Microsoft.Maui.Handlers.ViewHandler`2[[Microsoft.Maui.ISwipeItemView, Microsoft.Maui, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null],[Microsoft.Maui.Platform.ContentViewGroup, Microsoft.Maui, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]].SetVirtualView(IElement view) in /_/src/Core/src/Handlers/View/ViewHandlerOfT.cs:line 56
[mono-rt]    at Microsoft.Maui.Controls.Element.SetHandler(IElementHandler newHandler) in /_/src/Controls/src/Core/Element/Element.cs:line 1027
[mono-rt]    at Microsoft.Maui.Controls.Element.set_Handler(IElementHandler value) in /_/src/Controls/src/Core/Element/Element.cs:line 969
[mono-rt]    at Microsoft.Maui.Controls.VisualElement.Microsoft.Maui.IElement.set_Handler(IElementHandler value) in /_/src/Controls/src/Core/VisualElement/VisualElement.cs:line 2138
[mono-rt]    at Microsoft.Maui.Platform.ElementExtensions.ToHandler(IElement view, IMauiContext context) in /_/src/Core/src/Platform/ElementExtensions.cs:line 96
[mono-rt]    at Microsoft.Maui.Platform.ElementExtensions.ToPlatform(IElement view, IMauiContext context) in /_/src/Core/src/Platform/ElementExtensions.cs:line 127
[mono-rt]    at Microsoft.Maui.Platform.MauiSwipeView.UpdateSwipeItems() in /_/src/Core/src/Platform/Android/MauiSwipeView.cs:line 548
[mono-rt]    at Microsoft.Maui.Platform.MauiSwipeView.ProcessTouchMove(PointF point) in /_/src/Core/src/Platform/Android/MauiSwipeView.cs:line 396
[mono-rt]    at Microsoft.Maui.Platform.MauiSwipeView.HandleTouchInteractions(GestureStatus status, PointF point) in /_/src/Core/src/Platform/Android/MauiSwipeView.cs:line 362
[mono-rt]    at Microsoft.Maui.Platform.MauiSwipeView.ProcessSwipingInteractions(MotionEvent e) in /_/src/Core/src/Platform/Android/MauiSwipeView.cs:line 329
[mono-rt]    at Microsoft.Maui.Platform.MauiSwipeView.OnTouchEvent(MotionEvent e) in /_/src/Core/src/Platform/Android/MauiSwipeView.cs:line 86
[mono-rt]    at Android.Views.View.n_OnTouchEvent_Landroid_view_MotionEvent_(IntPtr jnienv, IntPtr native__this, IntPtr native_e) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/obj/Release/net9.0/android-35/mcw/Android.Views.View.cs:line 20212
[mono-rt]    at Android.Runtime.JNINativeWrapper.Wrap_JniMarshal_PPL_Z(_JniMarshal_PPL_Z callback, IntPtr jnienv, IntPtr klazz, IntPtr p0) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Android.Runtime/JNINativeWrapper.g.cs:line 132
[mono-rt]    at Java.Interop.JniEnvironment.InstanceMethods.CallNonvirtualBooleanMethod(JniObjectReference instance, JniObjectReference type, JniMethodInfo method, JniArgumentValue* args) in /Users/runner/work/1/s/xamarin-android/external/Java.Interop/src/Java.Interop/obj/Release/net8.0/JniEnvironment.g.cs:line 20459
[mono-rt]    at Java.Interop.JniPeerMembers.JniInstanceMethods.InvokeVirtualBooleanMethod(String encodedMember, IJavaPeerable self, JniArgumentValue* parameters) in /Users/runner/work/1/s/xamarin-android/external/Java.Interop/src/Java.Interop/Java.Interop/JniPeerMembers.JniInstanceMethods_Invoke.cs:line 164
[mono-rt]    at Android.Views.View.DispatchTouchEvent(MotionEvent e) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/obj/Release/net9.0/android-35/mcw/Android.Views.View.cs:line 16443
[mono-rt]    at Microsoft.Maui.Platform.MauiSwipeView.DispatchTouchEvent(MotionEvent e) in /_/src/Core/src/Platform/Android/MauiSwipeView.cs:line 207
[mono-rt]    at Android.Views.View.n_DispatchTouchEvent_Landroid_view_MotionEvent_(IntPtr jnienv, IntPtr native__this, IntPtr native_e) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/obj/Release/net9.0/android-35/mcw/Android.Views.View.cs:line 16430
[mono-rt]    at Android.Runtime.JNINativeWrapper.Wrap_JniMarshal_PPL_Z(_JniMarshal_PPL_Z callback, IntPtr jnienv, IntPtr klazz, IntPtr p0) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Android.Runtime/JNINativeWrapper.g.cs:line 132
[mono-rt]    at Java.Interop.JniEnvironment.InstanceMethods.CallNonvirtualBooleanMethod(JniObjectReference instance, JniObjectReference type, JniMethodInfo method, JniArgumentValue* args) in /Users/runner/work/1/s/xamarin-android/external/Java.Interop/src/Java.Interop/obj/Release/net8.0/JniEnvironment.g.cs:line 20459
[mono-rt]    at Java.Interop.JniPeerMembers.JniInstanceMethods.InvokeVirtualBooleanMethod(String encodedMember, IJavaPeerable self, JniArgumentValue* parameters) in /Users/runner/work/1/s/xamarin-android/external/Java.Interop/src/Java.Interop/Java.Interop/JniPeerMembers.JniInstanceMethods_Invoke.cs:line 164
[mono-rt]    at Android.App.Activity.DispatchTouchEvent(MotionEvent ev) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/obj/Release/net9.0/android-35/mcw/Android.App.Activity.cs:line 2136
[mono-rt]    at Microsoft.Maui.MauiAppCompatActivity.DispatchTouchEvent(MotionEvent e) in /_/src/Core/src/Platform/Android/MauiAppCompatActivity.cs:line 40
[mono-rt]    at Android.App.Activity.n_DispatchTouchEvent_Landroid_view_MotionEvent_(IntPtr jnienv, IntPtr native__this, IntPtr native_ev) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/obj/Release/net9.0/android-35/mcw/Android.App.Activity.cs:line 2123
[mono-rt]    at Android.Runtime.JNINativeWrapper.Wrap_JniMarshal_PPL_Z(_JniMarshal_PPL_Z callback, IntPtr jnienv, IntPtr klazz, IntPtr p0) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Android.Runtime/JNINativeWrapper.g.cs:line 132
[mono-rt]   --- End of managed Java.Lang.IllegalArgumentException stack trace ---

[mono-rt] java.lang.IllegalArgumentException: Cannot add a null child view to a ViewGroup
[mono-rt] 	at android.view.ViewGroup.addView(ViewGroup.java:5015)
[mono-rt] 	at android.view.ViewGroup.addView(ViewGroup.java:4997)
[mono-rt] 	at crc6452ffdc5b34af3a0f.MauiSwipeView.n_onTouchEvent(Native Method)
[mono-rt] 	at crc6452ffdc5b34af3a0f.MauiSwipeView.onTouchEvent(MauiSwipeView.java:62)
[mono-rt] 	at android.view.View.performOnTouchCallback(View.java:16447)
[mono-rt] 	at android.view.View.dispatchTouchEvent(View.java:16404)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3116)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2789)
[mono-rt] 	at crc6452ffdc5b34af3a0f.MauiSwipeView.n_dispatchTouchEvent(Native Method)
[mono-rt] 	at crc6452ffdc5b34af3a0f.MauiSwipeView.dispatchTouchEvent(MauiSwipeView.java:76)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3122)
[mono-rt] 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2803)
[mono-rt] 	at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:458)
[mono-rt] 	at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1980)
[mono-rt] 	at android.app.Activity.dispatchTouchEvent(Activity.java:4533)
[mono-rt] 	at crc6488302ad6e9e4df1a.MauiAppCompatActivity.n_dispatchTouchEvent(Native Method)
[mono-rt] 	at crc6488302ad6e9e4df1a.MauiAppCompatActivity.dispatchTouchEvent(MauiAppCompatActivity.java:53)
[mono-rt] 	at androidx.appcompat.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:69)
[mono-rt] 	at io.sentry.android.core.internal.gestures.WindowCallbackAdapter.dispatchTouchEvent(WindowCallbackAdapter.java:39)
[mono-rt] 	at io.sentry.android.core.internal.gestures.SentryWindowCallback.dispatchTouchEvent(SentryWindowCallback.java:64)
[mono-rt] 	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:416)
[mono-rt] 	at android.view.View.dispatchPointerEvent(View.java:16729)
[mono-rt] 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:7947)
[mono-rt] 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:7710)
[mono-rt] 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:7106)
[mono-rt] 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:7163)
[mono-rt] 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:7129)
[mono-rt] 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:7295)
[mono-rt] 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:7137)
[mono-rt] 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:7352)
[mono-rt] 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:7110)
[mono-rt] 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:7163)
[mono-rt] 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:7129)
[mono-rt] 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:7137)
[mono-rt] 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:7110)
[mono-rt] 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:10214)
[mono-rt] 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:10165)
[mono-rt] 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:10134)
[mono-rt] 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:10356)
[mono-rt] 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:295)
[mono-rt] 	at android.view.InputEventReceiver.nativeConsumeBatchedInputEvents(Native Method)
[mono-rt] 	at android.view.InputEventReceiver.consumeBatchedInputEvents(InputEventReceiver.java:259)
[mono-rt] 	at android.view.ViewRootImpl.doConsumeBatchedInput(ViewRootImpl.java:10313)
[mono-rt] 	at android.view.ViewRootImpl$ConsumeBatchedInputRunnable.run(ViewRootImpl.java:10449)
[mono-rt] 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1406)
[mono-rt] 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1415)
[mono-rt] 	at android.view.Choreographer.doCallbacks(Choreographer.java:1015)
[mono-rt] 	at android.view.Choreographer.doFrame(Choreographer.java:938)
[mono-rt] 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1389)
[mono-rt] 	at android.os.Handler.handleCallback(Handler.java:959)
[mono-rt] 	at android.os.Handler.dispatchMessage(Handler.java:100)
[mono-rt] 	at android.os.Looper.loopOnce(Looper.java:232)
[mono-rt] 	at android.os.Looper.loop(Looper.java:317)
[mono-rt] 	at android.app.ActivityThread.main(ActivityThread.java:8705)
[mono-rt] 	at java.lang.reflect.Method.invoke(Native Method)
[mono-rt] 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
[mono-rt] 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:886)
[mono-rt] 
[mono-rt]   --- End of managed Java.Lang.IllegalArgumentException stack trace ---
```

### Steps to Reproduce

_No response_

### Link to public reproduction project repository

_No response_

### Version with bug

9.0.10 SR1

### Is this a regression from previous behavior?

Yes, this used to work in .NET MAUI

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 15

### Did you find any workaround?

_No response_

### Relevant log output

```shell

```

I' also getting sometimes this exception:
```
[mono-rt] [ERROR] FATAL UNHANDLED EXCEPTION: System.InvalidOperationException: Operations that change non-concurrent collections must have exclusive access. A concurrent update was performed on this collection and corrupted its state. The collection's state is no longer correct.
[mono-rt]    at System.Collections.Generic.HashSet`1[[System.String, System.Private.CoreLib, Version=9.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].AddIfNotPresent(String value, Int32& location)
[mono-rt]    at System.Collections.Generic.HashSet`1[[System.String, System.Private.CoreLib, Version=9.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].Add(String item)
[mono-rt]    at Microsoft.Maui.Controls.Element.OnBindablePropertySet(BindableProperty property, Object original, Object value, Boolean changed, Boolean willFirePropertyChanged) in /_/src/Controls/src/Core/Element/Element.cs:line 639
[mono-rt]    at Microsoft.Maui.Controls.BindableObject.ClearValueCore(BindableProperty property, SetterSpecificity specificity) in /_/src/Controls/src/Core/BindableObject.cs:line 151
[mono-rt]    at Microsoft.Maui.Controls.BindableObject.ClearValue(BindableProperty property, SetterSpecificity specificity) in /_/src/Controls/src/Core/BindableObject.cs:line 107
[mono-rt]    at Microsoft.Maui.Controls.Setter.UnApply(BindableObject target, SetterSpecificity specificity) in /_/src/Controls/src/Core/Setter.cs:line 104
[mono-rt]    at Microsoft.Maui.Controls.VisualStateManager.GoToState(VisualElement visualElement, String name) in /_/src/Controls/src/Core/VisualStateManager.cs:line 105
[mono-rt]    at Microsoft.Maui.Controls.VisualElement.ChangeVisualState() in /_/src/Controls/src/Core/VisualElement/VisualElement.cs:line 1582
[mono-rt]    at Microsoft.Maui.Controls.View.ChangeVisualState() in /_/src/Controls/src/Core/View/View.cs:line 204
[mono-rt]    at Microsoft.Maui.Controls.VisualElement.OnIsEnabledPropertyChanged(BindableObject bindable, Object oldValue, Object newValue) in /_/src/Controls/src/Core/VisualElement/VisualElement.cs:line 1646
[mono-rt]    at Microsoft.Maui.Controls.BindableObject.OnBindablePropertySet(BindableProperty property, Object original, Object value, Boolean didChange, Boolean willFirePropertyChanged) in /_/src/Controls/src/Core/BindableObject.cs:line 675
[mono-rt]    at Microsoft.Maui.Controls.Element.OnBindablePropertySet(BindableProperty property, Object original, Object value, Boolean changed, Boolean willFirePropertyChanged) in /_/src/Controls/src/Core/Element/Element.cs:line 642
[mono-rt]    at Microsoft.Maui.Controls.BindableObject.SetValueActual(BindableProperty property, BindablePropertyContext context, Object value, Boolean currentlyApplying, SetValueFlags attributes, SetterSpecificity specificity, Boolean silent) in /_/src/Controls/src/Core/BindableObject.cs:line 662
[mono-rt]    at Microsoft.Maui.Controls.BindableObject.SetValueCore(BindableProperty property, Object value, SetValueFlags attributes, SetValuePrivateFlags privateAttributes, SetterSpecificity specificity) in /_/src/Controls/src/Core/BindableObject.cs:line 590
[mono-rt]    at Microsoft.Maui.Controls.BindableObject.SetValue(BindableProperty property, Object value) in /_/src/Controls/src/Core/BindableObject.cs:line 495
[mono-rt]    at Microsoft.Maui.Controls.VisualElement.set_IsEnabled(Boolean value) in /_/src/Controls/src/Core/VisualElement/VisualElement.cs:line 632
[mono-rt]    at MyApp.Forms.Controls.SwipeItemViewIcon.OnSwipeChangingAsync(Double threshold, Double offset) in D:\Dev\MyApp.Forms\Controls\SwipeItemViewIcon.xaml.cs:line 45
[mono-rt]    at MyApp.Forms.Controls.SwipeViewBase.OnSwipeChanging(Object sender, SwipeChangingEventArgs e) in D:\Dev\MyApp.Forms\Controls\SwipeViewBase.cs:line 73
[mono-rt]    at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
[mono-rt]    at Android.App.SyncContext.<>c__DisplayClass2_0.<Post>b__0() in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Android.App/SyncContext.cs:line 36
[mono-rt]    at Java.Lang.Thread.RunnableImplementor.Run() in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Java.Lang/Thread.cs:line 37
[mono-rt]    at Java.Lang.IRunnableInvoker.n_Run(IntPtr jnienv, IntPtr native__this) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/obj/Release/net9.0/android-35/mcw/Java.Lang.IRunnable.cs:line 59
[mono-rt]    at Android.Runtime.JNINativeWrapper.Wrap_JniMarshal_PP_V(_JniMarshal_PP_V callback, IntPtr jnienv, IntPtr klazz) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Android.Runtime/JNINativeWrapper.g.cs:line 22
```

--- a/src/Controls/tests/TestCases.HostApp/Issues/Issue26328.xaml
+++ b/src/Controls/tests/TestCases.HostApp/Issues/Issue26328.xaml
@@ -1,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<controls:TestContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+			       xmlns:controls="clr-namespace:Maui.Controls.Sample.Issues"
+             xmlns:cmp="clr-namespace:Microsoft.Maui.Controls.Compatibility;assembly=Microsoft.Maui.Controls"
+             x:Class="Maui.Controls.Sample.Issues.Issue26328">
+  <ContentPage.Content>
+    <CollectionView ItemsSource="{Binding ItemList}">
+      <CollectionView.ItemTemplate>
+        <DataTemplate x:DataType="controls:Issue26328ItemModel">
+          <SwipeView>
+            <Border Padding="10,30">
+              <Label>
+                <Label.Text>
+                  <MultiBinding Converter="{controls:Issue26328TestConverter}">
+                    <Binding x:DataType="controls:Issue26328ItemModel" Path="Id" />
+                    <Binding x:DataType="controls:Issue26328ItemModel" Path="Title" />
+                  </MultiBinding>
+                </Label.Text>
+              </Label>
+            </Border>
+            <SwipeView.RightItems>
+              <SwipeItems Mode="Execute">
+                <SwipeItemView Command="{Binding SwipeCommand}">
+                  <ContentView Padding="10"
+                               HorizontalOptions="End"
+                               VerticalOptions="Center">
+                    <Label Text="X" />
+                  </ContentView>
+                </SwipeItemView>
+              </SwipeItems>
+            </SwipeView.RightItems>
+          </SwipeView>
+        </DataTemplate>
+      </CollectionView.ItemTemplate>
+    </CollectionView>
+  </ContentPage.Content>
+</controls:TestContentPage>

--- a/src/Controls/tests/TestCases.HostApp/Issues/Issue26328.xaml
+++ b/src/Controls/tests/TestCases.HostApp/Issues/Issue26328.xaml
@@ -5,7 +5,8 @@
              xmlns:cmp="clr-namespace:Microsoft.Maui.Controls.Compatibility;assembly=Microsoft.Maui.Controls"
              x:Class="Maui.Controls.Sample.Issues.Issue26328">
   <ContentPage.Content>
-    <CollectionView ItemsSource="{Binding ItemList}">
+    <CollectionView AutomationId="TestCollectionView" 
+                    ItemsSource="{Binding ItemList}">
       <CollectionView.ItemTemplate>
         <DataTemplate x:DataType="controls:Issue26328ItemModel">
           <SwipeView>

--- a/src/Controls/tests/TestCases.HostApp/Issues/Issue26328.xaml.cs
+++ b/src/Controls/tests/TestCases.HostApp/Issues/Issue26328.xaml.cs
@@ -1,0 +1,99 @@
+﻿using System.Collections.ObjectModel;
+using System.Globalization;
+
+namespace Maui.Controls.Sample.Issues
+{
+	[Issue(IssueTracker.Github, 26328, "SwipeView causes Java.Lang.IllegalArgumentException: Cannot add a null child view to a ViewGroup", PlatformAffected.Android)]
+	public partial class Issue26328 : TestContentPage
+	{
+		public Issue26328()
+		{
+			InitializeComponent();
+		}
+
+		protected override void Init()
+		{
+			BindingContext = new Issue26328ViewModel();
+		}
+	}
+
+	public class Issue26328ItemModel
+	{
+		public int Id { get; set; }
+		public string Title { get; set; }
+
+		public Issue26328ItemModel(int i)
+		{
+			Id = i;
+			Title = "Hello";
+		}
+
+		public Command SwipeCommand =>
+			new Command(DeleteMessage);
+
+		void DeleteMessage()
+		{
+#pragma warning disable CS0618 // Type or member is obsolete
+			MessagingCenter.Send<Issue26328ItemModel, Issue26328RemoveMessage>(this, "Swipe", new Issue26328RemoveMessage(this));
+#pragma warning disable CS0618 // Type or member is obsolete
+		}
+	}
+
+	public class Issue26328RemoveMessage
+	{
+		public Issue26328ItemModel Item { get; }
+
+		public Issue26328RemoveMessage(Issue26328ItemModel item)
+		{
+			Item = item;
+		}
+	}
+
+	public class Issue26328ViewModel
+	{
+		public ObservableCollection<Issue26328ItemModel> ItemList { get; set; }
+
+		public Issue26328ViewModel()
+		{
+#pragma warning disable CS0618 // Type or member is obsolete
+			MessagingCenter.Subscribe<Issue26328ItemModel, Issue26328RemoveMessage>(this, "Swipe", (sender, arg) => RemoveItem(arg.Item));
+#pragma warning disable CS0618 // Type or member is obsolete
+			ItemList = [];
+			for (var i = 0; i < 200; ++i)
+			{
+				ItemList.Add(new Issue26328ItemModel(i));
+			}
+		}
+
+		void RemoveItem(Issue26328ItemModel item)
+		{
+			ItemList.Remove(item);
+			GC.Collect();
+		}
+	}
+
+	[AcceptEmptyServiceProvider]
+	public class Issue26328TestConverter : IMultiValueConverter, IMarkupExtension
+	{
+		public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+		{
+			if (values[0] == null || values[1] == null)
+			{
+				return null;
+			}
+			var id = (int)values[0];
+			var title = (string)values[1];
+			return $"{title} - {id}";
+		}
+
+		public object[] ConvertBack(object value, Type[] targetTypes, object parameter, CultureInfo culture)
+		{
+			throw new NotImplementedException();
+		}
+
+		public object ProvideValue(IServiceProvider serviceProvider)
+		{
+			return this;
+		}
+	}
+}

--- a/src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue26328.cs
+++ b/src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue26328.cs
@@ -1,0 +1,31 @@
+﻿#if ANDROID // Crash only happened on Android
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue26328 : _IssuesUITest
+	{ 
+		public override string Issue => "SwipeView causes Java.Lang.IllegalArgumentException: Cannot add a null child view to a ViewGroup";
+
+		public Issue26328(TestDevice device)
+		: base(device)
+		{ }
+
+		[Test]
+		[Category(UITestCategories.SwipeView)]
+		public void NoCrashRemovingSwipeItems()
+		{
+			App.WaitForElement("TestCollectionView");
+
+			for(int i = 0; i < 10; i++)
+			{
+				App.SwipeRightToLeft();
+				App.ScrollDown("TestCollectionView", ScrollStrategy.Gesture, swipePercentage: 0.2);
+				App.ScrollUp("TestCollectionView", ScrollStrategy.Gesture, swipePercentage: 0.2);
+			}
+		}
+	}
+}
+#endif

--- a/src/Core/src/Handlers/SwipeItemView/SwipeItemViewHandler.Android.cs
+++ b/src/Core/src/Handlers/SwipeItemView/SwipeItemViewHandler.Android.cs
@@ -57,6 +57,7 @@ namespace Microsoft.Maui.Handlers
 		protected override void DisconnectHandler(ContentViewGroup platformView)
 		{
 			// If we're being disconnected from the xplat element, then we should no longer be managing its children
+			platformView.CrossPlatformLayout = null;
 			platformView.RemoveAllViews();
 			base.DisconnectHandler(platformView);
 		}

--- a/src/Core/src/Platform/Android/MauiSwipeView.cs
+++ b/src/Core/src/Platform/Android/MauiSwipeView.cs
@@ -526,7 +526,7 @@ namespace Microsoft.Maui.Platform
 
 		void UpdateSwipeItems()
 		{
-			if (_contentView == null || _actionView != null)
+			if (_contentView == null || _contentView.IsDisposed() || _actionView != null)
 				return;
 
 			ISwipeItems? items = GetSwipeItemsByDirection();
@@ -545,22 +545,25 @@ namespace Microsoft.Maui.Platform
 
 			foreach (var item in items)
 			{
-				AView swipeItem = item.ToPlatform(MauiContext);
+				AView? swipeItem = item?.ToPlatform(MauiContext);
 
-				if (item is ISwipeItemView formsSwipeItemView)
+				if (swipeItem is not null)
 				{
-					_actionView.AddView(swipeItem);
-					UpdateSwipeItemViewLayout(formsSwipeItemView);
-					_swipeItems.Add(formsSwipeItemView, swipeItem);
-				}
-				else if (item is ISwipeItemMenuItem menuItem)
-				{
-					_actionView.AddView(swipeItem);
-					_swipeItems.Add(item, swipeItem);
-				}
+					if (item is ISwipeItemView formsSwipeItemView)
+					{
+						_actionView.AddView(swipeItem);
+						UpdateSwipeItemViewLayout(formsSwipeItemView);
+						_swipeItems.Add(formsSwipeItemView, swipeItem);
+					}
+					else if (item is ISwipeItemMenuItem menuItem)
+					{
+						_actionView.AddView(swipeItem);
+						_swipeItems.Add(item, swipeItem);
+					}
 
-				if (swipeItem != null)
-					swipeItems.Add(swipeItem);
+					if (swipeItem != null)
+						swipeItems.Add(swipeItem);
+				}
 			}
 
 			AddView(_actionView);

